application/controllers/UploaderController.php

```diff
@@ -282,7 +282,7 @@ public function run($actionID)
         }
         /* No action */
         $meta = '';
-        App()->getClientScript()->registerPackage('jquery');
+        //App()->getClientScript()->registerPackage('jquery');
```

Comment:
Please either remove line or add a comment on why it's commented out, thank you!

Reply:
Fixed! Sorry.